app/controllers/nodes_controller.rb

```diff
@@ -18,18 +18,34 @@ def edit
   # POST /nodes
   def create
     @node.label = 'unnamed' unless @node.label.present?
-    if @node.save
-      track_created(@node)
-      flash[:notice] = 'Successfully created node.'
-      redirect_to @node
-    else
-      parent = @node.parent
-      if parent
-        redirect_to parent, alert: @node.errors.full_messages.join('; ')
-      else
-        redirect_to summary_path, alert: @node.errors.full_messages.join('; ')
+    @node.save!
+    track_created(@node)
+    flash[:notice] = 'Successfully created node.'
+    redirect_to @node
+  end
+
+  def multiple
```

Comment:
Naming convention? Maybe `#create_multiple` is more descriptive? I don't mind which one we agree on, but I'd rather it be consistent across controllers.

See:
https://github.com/dradis/dradis-ce/blob/add-multiple-nodes/app/controllers/evidence_controller.rb#L39